x-pack/platform/plugins/shared/fleet/public/applications/fleet/layouts/default/default.tsx

```diff
@@ -37,6 +37,8 @@ export const DefaultLayout: React.FunctionComponent<Props> = ({
   const { docLinks } = useStartServices();
   const granularPrivilegesCallout = useDismissableTour('GRANULAR_PRIVILEGES');
   const { enableAutomaticAgentUpgrades } = ExperimentalFeaturesService.get();
+  const canEnableAutomaticAgentUpgrades =
```

Comment:
should we extract `canEnableAutomaticAgentUpgrades` as a function to reuse, similar to https://github.com/elastic/kibana/blob/18f606e2fa72abd4c399b4accdd617d06853dccd/x-pack/platform/plugins/shared/fleet/server/services/setup/fleet_synced_integrations.ts#L68?